python/cudf_polars/tests/test_string.py

```diff
@@ -35,6 +35,15 @@ def test_contains(substr):
     assert_gpu_result_equal(query)
 
 
+def test_contains_column():
+    ldf = pl.DataFrame(
+        {"a": ["AbC", "de", "FGHI", "j", "kLm", "nOPq", None, "RsT", None, "uVw"]}
+    ).lazy()
+
+    query = ldf.select(pl.col("a").str.contains(pl.col("a"), literal=True))
+    assert_gpu_result_equal(query)
```

Comment:
Please also add coverage for the case that the pattern is a literal value:

```
pl.lit("GH")
``` 
(say)